dracut/99emergency-timeout/ignition-virtio-dump-journal.sh

```diff
@@ -3,6 +3,13 @@ set -euo pipefail
 
 port=/dev/virtio-ports/com.coreos.ignition.journal
 if [ -e "${port}" ]; then
+    # wait for the journal to wind down
+    for i in {1..15}; do
+        if journalctl -q -u systemd-journald.service --grep 'Journal stopped'; then
```

Comment:
In https://github.com/coreos/fedora-coreos-config/pull/339/files#diff-6b1b11642c8646b9cd8e6704c1896ae2R27 I tried doing this in a non-polling way by doing `| head -1` but I think that has bugs unfortunately.